test/polyline.test.js

```diff
@@ -1,10 +1,32 @@
 var assert = require('assert'),
-    polyline = require('../');
+    polyline = require('../'),
+    package = require('../package.json');
 
 describe('polyline', function() {
     var example = [[38.5, -120.2], [40.7, -120.95], [43.252, -126.453]],
         // encoded value will enclude slashes -> tests escaping
-        example_slashes = [[35.6,-82.55], [35.59985, -82.55015], [35.6,-82.55]];
+        example_slashes = [[35.6,-82.55], [35.59985, -82.55015], [35.6,-82.55]],
+        example_flipped = [[-120.2, 38.5], [-120.95, 40.7], [-126.453, 43.252]],
+        dependencies = { "turf-flip": "*", "geojson-coords": "*" }
+        geojson = {
+            "type": "Feature",
+            "properties": {},
+            "geometry": {
+                "type": "Point",
+                "coordinates": [20.566406, 43.421008]
+            }
+        },
+        flipped_coords = [43.421008, 20.566406];
+
+    describe('#dependencies', function() {
```

Comment:
Ah yeah, those were fixtures for tests below to ensure dependencies. I was really going all out, trying to test every little thing, because I have no idea what I'm doing 😉